### ISSUE TITLE
Netopeer2 installation fails due to problem in sysrepo installation

### ISSUE THREAD

**Issue by anavajasc** [sysrepo/sysrepo#2980]
Hi,
I am trying to run the netopeer2 and finding a problem in sysrepo.
Install the project...
-- Install configuration: "Debug"
-- Up-to-date: /usr/local/share/yang/modules/netopeer2
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-tcp-server@2019-07-02.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-netconf-monitoring@2010-10-04.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/notifications@2008-07-14.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-yang-push@2019-09-09.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/iana-crypt-hash@2014-08-06.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-netconf-acm@2018-02-14.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-x509-cert-to-name@2014-12-10.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-ssh-server@2019-07-02.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-ip@2018-02-22.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-keystore@2019-07-02.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-restconf@2017-01-26.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-netconf@2013-09-29.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-interfaces@2018-02-20.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-netconf-nmda@2019-01-07.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-crypto-types@2019-07-02.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-ssh-common@2019-07-02.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-subscribed-notifications@2019-09-09.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-network-instance@2019-01-21.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-tcp-client@2019-07-02.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-tcp-common@2019-07-02.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-truststore@2019-07-02.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-yang-schema-mount@2019-01-14.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-netconf-server@2019-07-02.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/nc-notifications@2008-07-14.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-datastores@2018-02-14.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-yang-patch@2017-02-22.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-tls-server@2019-07-02.yang
-- Up-to-date: /usr/local/share/yang/modules/netopeer2/ietf-tls-common@2019-07-02.yang
-- Up-to-date: /usr/local/share/netopeer2
-- Up-to-date: /usr/local/share/netopeer2/remove.sh
-- Up-to-date: /usr/local/share/netopeer2/merge_hostkey.sh
-- Up-to-date: /usr/local/share/netopeer2/setup.sh
-- Up-to-date: /usr/local/share/netopeer2/merge_config.sh
-- Up-to-date: /usr/local/sbin/netopeer2-server
-- Up-to-date: /usr/local/share/man/man8/netopeer2-server.8
-- Installing missing sysrepo modules...
/usr/local/bin/sysrepoctl: error while loading shared libraries: libsysrepo.so.7: cannot open shared object file: No such file or directory
/usr/local/bin/sysrepoctl: error while loading shared libraries: libsysrepo.so.7: cannot open shared object file: No such file or directory
CMake Error at cmake_install.cmake:99 (message):
   scripts/setup.sh failed: 127

I am new in Netopeer2, can you give me some suggestions.

Regards,
Alvaro.

**Comment by myonlystarWang** [comment]
Did you run `ldconfig `first after `sysrepo `installed?

**Comment by anavajasc** [comment]
No, I dind't install ldconfig. I have deleted and cloned again the sysrepo and now after the installation I have installed the ldconfig and the netopeer2 is installed correctly. Thank very much for the help!
https://hub.docker.com/r/sysrepo/sysrepo/dockerfile (it help me for the sysrepo installation)
